modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/GoClientCodegen.java

```diff
@@ -56,8 +56,7 @@ public GoClientCodegen() {
                 "case", "defer", "go", "map", "struct",
                 "chan", "else", "goto", "package", "switch",
                 "const", "fallthrough", "if", "range", "type",
-                "continue", "for", "import", "return", "var", "error", "ApiResponse",
-                "nil")
+                "continue", "for", "import", "return", "var", "error", "ApiResponse")
```

Comment:
@wing328 no, It's my fault I was using that code to get inspired and write the Eiffel one.
I will fix it ASAP.
Thanks